packages/manager/cypress/e2e/core/account/users-landing-page.spec.ts

```diff
@@ -0,0 +1,600 @@
+      username: 'unrestricted-user',
+      restricted: false,
+    });
+
+    const mockRestrictedUser = accountUserFactory.build({
+      username: 'restricted-user',
+      restricted: false,
+    });
+
+    const mockChildUser = accountUserFactory.build({
+      username: 'child-user',
+      restricted: false,
+      user_type: 'child',
+    });
+
+    const mockProxyUser = accountUserFactory.build({
+      username: 'proxy-user',
+      restricted: false,
+      user_type: 'proxy',
+    });
+
+    const mockUsers = [
+      mockUser,
+      mockRestrictedUser,
+      mockChildUser,
+      mockProxyUser,
+    ];
+
+    const mockUserGrants = grantsFactory.build({
+      global: { child_account_access: true },
+    });
+
+    // TODO: Parent/Child - M3-7559 clean up when feature is live in prod and feature flag is removed.
+    mockAppendFeatureFlags({
+      parentChildAccountAccess: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Initially mock user with unrestricted account access.
+    mockGetUsers(mockUsers).as('getUsers');
+    mockUsers.forEach((user) => {
+      mockGetUser(user);
+      mockGetUserGrantsUnrestrictedAccess(user.username);
+    });
+    mockGetUserGrants(mockUser.username, mockUserGrants);
+    mockGetProfile(mockProfile);
+
+    // Navigate to Users & Grants page.
+    cy.visitWithLogin('/account/users');
+    cy.wait('@getUsers');
+
+    // Confirm that "Child account access" column is present
+    cy.findByText('Child Account Access').should('be.visible');
+    mockUsers.forEach((user) => {
+      cy.get(`[aria-label="User ${user.username}"]`)
+        .should('be.visible')
+        .within(() => {
+          if (user === mockUser) {
+            // The status should be "Enabled" when the user with "child_account_access" grant
+            cy.findByText('Enabled').should('be.visible');
+          } else {
+            // Others should be "Disabled"
+            cy.findByText('Disabled').should('be.visible');
+          }
+        });
+    });
+  });
+
+  /*
+   * Confirm that "Business Partner Settings" section is not present for parent users
+   */
+  it('hides "Business Partner Settings" section for parent users', () => {
+    const mockProfile = profileFactory.build({
+      username: 'unrestricted-parent-user',
+      restricted: false,
+      user_type: 'parent',
+    });
+
+    const mockUser = accountUserFactory.build({
+      username: 'unrestricted-user',
+      restricted: false,
+    });
+
+    // TODO: Parent/Child - M3-7559 clean up when feature is live in prod and feature flag is removed.
+    mockAppendFeatureFlags({
+      parentChildAccountAccess: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Initially mock user with unrestricted account access.
+    mockGetUsers([mockUser]).as('getUsers');
+    mockGetUser(mockUser).as('getUser');
```

Comment:
```suggestion
    mockGetUser(mockUser);
```
Can we clean up the aliases whenever they're not used? - looks like we've got several unused throughout this file.